NitroxClient/Communication/Packets/Processors/BulkheadDoorStateChangedProcessor.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxClient.MonoBehaviours.CinematicController;
+using NitroxClient.Unity.Helper;
+using UnityEngine;
+using UWE;
+
+public class BulkheadDoorStateChangedProcessor : ClientPacketProcessor<BulkheadDoorStateChanged>
+{
+
+    private readonly PlayerManager remotePlayerManager;
+
+    public BulkheadDoorStateChangedProcessor(PlayerManager remotePlayerManager)
+    {
+        this.remotePlayerManager = remotePlayerManager;
+    }
+
+
+    public override void Process(BulkheadDoorStateChanged packet)
+    {
+        if (!NitroxEntity.TryGetObjectFrom(packet.Id, out GameObject bulkheadDoor))
+        {
+            Log.Error($"[BulkheadDoorStateChangedProcessor] Couldn't find GameObject for {packet.Id}");
+            return;
+        }
+
+        if (!remotePlayerManager.TryFind(packet.PlayerId, out RemotePlayer otherPlayer))
+        {
+            Log.Error($"[BulkheadDoorStateChangedProcessor] Couldn't find {nameof(RemotePlayer)} for {packet.PlayerId}");
+            return;
+        }
+
+        if (!bulkheadDoor.TryGetComponentInChildren(out BulkheadDoor door))
+        {
+            Log.Info("[BulkheadDoorStateChangedProcessor] Unable to find BulkheadDoor");
+            return;
+        }
+
+        bool prevState = !packet.IsOpen;
+
+        door.animator.SetBool("player_in_front", packet.IsFacingDoor);
+
+        PlayerCinematicController cinematicController = GetPlayerCinematicController(door, prevState, packet.IsFacingDoor);
+        MultiplayerCinematicController multiPlayerCinematicController = MultiplayerCinematicController.Initialize(cinematicController);
```

Comment:
`Initialize` should not be called every CallStartCinematicMode call. Look at `NitroxClient.Communication.Packets.Processors.PlayerCinematicControllerCallProcessor.cs` for reference.